src/elements/content-sidebar/activity-feed/common/activity-message/ActivityMessage.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../../../../styles/variables';
+
+.bcs-ActivityMessage {
+    white-space: pre-wrap;
+
+    .link {
+        color: $bdl-box-blue;
+        white-space: normal;
+
+        &:hover {
```

Comment:
Is `&:focus` also needed here?

Reply:
This is what was there, just moved to the new file. Possibly?